.travis.yml

```diff
@@ -22,11 +22,11 @@ script:
       export CARGO_INCREMENTAL=1 &&
       (test "$TRAVIS_RUST_VERSION" == 1.26.0 || rm -f Cargo.lock) &&
       cargo build --all &&
-      cargo build --all --all-features &&
-      (test "$TRAVIS_RUST_VERSION" == 1.26.0 || cargo test --all) &&
-      (test "$TRAVIS_RUST_VERSION" == 1.26.0 || cargo test --all --all-features) &&
+      (test "$TRAVIS_RUST_VERSION" == 1.26.0 || test "$TRAVIS_RUST_VERSION" == "stable" || cargo build --all --all-features) &&
+      (test "$TRAVIS_RUST_VERSION" == 1.26.0 || test "$TRAVIS_RUST_VERSION" == "stable" || cargo test --all) &&
+      (test "$TRAVIS_RUST_VERSION" == 1.26.0 || test "$TRAVIS_RUST_VERSION" == "stable" || cargo test --all --all-features) &&
       cargo doc --no-deps &&
-      (test "$TRAVIS_RUST_VERSION" == 1.26.0 || cargo clippy --all --tests -- --deny clippy::all) &&
+      (test "$TRAVIS_RUST_VERSION" == 1.26.0 || test "$TRAVIS_RUST_VERSION" == "stable" || cargo clippy --all --tests -- --deny clippy::all) &&
       (test "$TRAVIS_RUST_VERSION" == 1.26.0 || cargo fmt)
```

Comment:
I guess it makes sense to disable the tests on stable right now. But I'd like to wait for the 1.39 / tokio 0.2 release before actually putting this into master and by then we need to not forget re-enabling them.